docs/customization.md

```diff
@@ -60,6 +60,22 @@ def hijack_staff_other_staff(*, hijacker, hijacked):
 
     if hijacker.is_staff and not hijacked.is_superuser:
         return True
+
+
+# Add permissions in YourModel.Meta.permissions
+# See: https://docs.djangoproject.com/en/dev/topics/auth/customizing/#custom-permissions
+def hijack_allowed_group(*, hijacker: User, hijacked: User):
+    """Check if hijacker is a member of custom permission group belonging to a model in my_app."""
```

Comment:
Docstrings should describe the external behavior and be imperative. Something like:
```suggestion
    """Users with object based permissions or superusers may hijack any active user."""
```